projects/kubernetes/boot.sh

```diff
@@ -15,7 +15,8 @@ set -e
 : ${KUBE_RUN_ARGS:=}
 : ${KUBE_EFI:=}
 : ${KUBE_MAC:=}
-: ${KUBE_PRESERVE_STATE:=}
+: ${KUBE_PRESERVE_STATE:=true}
```

Comment:
With this change how can I now disable this? (I don't think I can). I think it would be best to invert the name and leaving it defaulting to blank, i.e. `KUBE_RESET_STATE`,  `KUBE_NUKE_STATE` or `KUBE_CLEAR_STATE` or somesuch.

Reply:
done